scripts/minikube-minishift-setup-env.sh

```diff
@@ -6,51 +6,48 @@ shout() {
   set -x
 }
 
-# Create a bin directory whereever script runs. This will be where all binaries that need to be in PATH will reside.
-export HOME=`pwd`/home
-export GOPATH="`pwd`/home/go"
-export GOBIN="$GOPATH/bin"
-mkdir -p $GOBIN
 # This si one of the variables injected by ci-firewall. Its purpose is to allow scripts to handle uniqueness as needed
 SCRIPT_IDENTITY=${SCRIPT_IDENTITY:-"def-id"}
 
-# Add GOBIN which is the bin dir we created earlier to PATH so any binaries there are automatically available in PATH
-export PATH=$PATH:$GOBIN
+setup_operator() {
+  SETUP_OPERATORS="./scripts/configure-cluster/common/setup-operators.sh"
 
-# Prep for integration/e2e
-shout "Building odo binaries"
-make bin
+  # The OLM Version
+  export OLM_VERSION="v0.17.0"
```

Comment:
We can (should?) use newer version now. 0.19.1 is available. We had earlier kept this at 0.17.0 due to issues with 0.18.2.

Reply:
Another way to not hard code this in script and pull artefacts from latest could be to do:
```sh
LATEST_RELEASE=$(curl -L -s -H 'Accept: application/json' https://github.com/operator-framework/operator-lifecycle-manager/releases/latest)
LATEST_VERSION=$(echo $LATEST_RELEASE | sed -e 's/.*"tag_name":"\([^"]*\)".*/\1/')
```
Now `echo $LATEST_VERSION` will have the value. I found this from https://blog.markvincze.com/download-artifacts-from-a-latest-github-release-in-sh-and-powershell/.

Reply:
Understood! I will update this accordingly
Thanks